CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ Note: This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0
 
 All notable changes to this project will be documented in this file.
 
+## [2.3.0.pre.1] - UNRELEASED
```

Comment:
You can test this gem in web with this branch: https://github.com/Invoca/web/tree/TECH-5255_testing_web_against_aggregate_saved_changes